news/fips-cve.md

```diff
@@ -0,0 +1,41 @@
+---
+breadcrumb: CVEs and FIPS
+---
+# CVEs and the FIPS provider
+
+After the release of OpenSSL 3.0.0, several CVEs have been identified
+and resolved.  While the majority of these vulnerabilities are unrelated
+to the validated FIPS provider, a few of them are applicable.  This table
+lists all of the CVEs issued since the FIPS provider's release and their
+relevance to it:
+
+
+CVE ID | Fixed | Impacted | Notes
```

Comment:
Perhaps use `FIPS Provider Impact` instead of just `Impacted`

Reply:
I don't want this one being too long.  `FIPS?` might be better.
Once we've multiple FIPS providers validated, I'd like to list versions here.  It's pointless doing so with only one.

Reply:
This whole file is about the FIPS provider...  not sure I understand what's unclear with "Impacted" in that case.  But of well, it was changed.